File: lab 1/lab1/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab1
{
    internal class Item
    {
        public int Id;
        public int Value;
        public int Weight;


        public Item(int id, int value, int weight)
        {
            Id = id;
            Value = value;
            Weight = weight;
        }

        public override string ToString()
        {
            return $"Id: {Id} V: {Value} W: {Weight}";
        }
    }

}

File: lab 5/src/main/java/org/example/Item.java
package org.example;

public class Item {
    public int id;
    public int value;
    public int weight;

    // Konstruktor
    public Item(int id, int value, int weight) {
        this.id = id;
        this.value = value;
        this.weight = weight;
    }

    @Override
    public String toString() {
        return "No: " + id + " v: " + value + " w: " + weight;
    }
}